CONTRIBUTORS.txt

```diff
@@ -78,6 +85,7 @@ needs to be listed here.
 - Tim Condon <0xtimc@gmail.com>
```

Comment:
@siemensikkema  if you would like me to add a proper email address here, please reply with the preferred email address.

Reply:
Ok, thanks! s.h.sikkema@gmail.com please :)

Reply:
thanks, done